plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/KtLintSupportedVersionsTest.kt

```diff
@@ -173,17 +173,9 @@ class KtLintSupportedVersionsTest : AbstractPluginTest() {
 
     class SupportedKtlintVersionsProvider : GradleArgumentsProvider() {
         private val supportedKtlintVersions = mutableListOf(
-            "0.47.1",
-            "0.48.2",
-            // "0.49.0" did not expose needed baseline classes
-            "0.49.1",
-            "0.50.0",
-            "1.0.1",
-            "1.1.1",
-            "1.2.1",
-            "1.3.1",
-            "1.4.1",
-            "1.5.0"
+            "1.0.1", // lowest
+            "1.5.0", // default
+            "1.6.0" // highest
```

Comment:
Should we keep a few more of these in here?

Reply:
the post-1.0 API and behavior has been very stable, so it seemed a bit redundant to test _every_ minor version, but this is just a build performance improvement. I have no problem with adding more. what do you think?  all of them again, or just a bigger subset?